cpp/src/arrow/util/async_util_test.cc

```diff
@@ -437,8 +437,8 @@ TEST(AsyncTaskScheduler, Priority) {
   constexpr int kNumConcurrentTasks = 8;
   std::unique_ptr<AsyncTaskScheduler::Throttle> throttle =
       AsyncTaskScheduler::MakeThrottle(kNumConcurrentTasks);
-  std::unique_ptr<AsyncTaskScheduler> task_group = AsyncTaskScheduler::Make(
-      throttle.get(), std::make_unique<PriorityQueue>());
+  std::unique_ptr<AsyncTaskScheduler> task_group =
```

Comment:
Can we use `auto` here?

Reply:
Done. I didn't change this file though, only formatted it.